README.md

```diff
@@ -2,6 +2,26 @@
 
 Reading data from package contents and files.
 
+A resource is data that can be read into a Dart program at runtime.
+A resource is identified by a URI. It can be loaded as bytes or data.
+The resource URI may be a package: URI.
+
+Example:
+
+```
+import 'package:resource/resource.dart';
```

Comment:
Technically you should also import dart:collection to have UTF8 for this to be a compete example. I'm not sure if that's important or not.